### DISCUSSION TITLE
Add "camera" as option for homepage

### DISCUSSION THREAD

**Discussion by Tuporat** [deckerst/aves#1026] [Ideas]
Hello

Firstly, many thanks for the great app.

One improvement I would like to see is the option to activate “camera” as the default start page. Unfortunately, I can currently only select “all”, “folder” or “tags”.

Many thanks and best regards

**Comment by deckerst** [comment]
It is already possible to set any filtered collection as home, using the `Collection` page top right menu > `Set as home`.
